### ISSUE TITLE
Error loading

### ISSUE THREAD

**Issue by sinsmin** [johnno1962/injectionforxcode#251]
Error loading /Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection:  dlopen(/Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection, 265): Symbol not found: __T0SPyxGs8_PointersWP
  Referenced from: /Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection
  Expected in: /Users/xxxx/Library/Developer/CoreSimulator/Devices/921C0B79-E1D9-45EA-9C55-BAF532EBBE2A/data/Containers/Bundle/Application/19B3DE92-2C96-4CB5-B394-2308819F5218/xxxxxx.app/Frameworks/libswiftCore.dylib
 in /Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection

**Comment by TPQuietBro** [comment]
I have the same question:
MyInjectionDemo[2319:97348] Error loading /Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection:  dlopen(/Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection, 265): Library not loaded: @rpath/XCTest.framework/XCTest
  Referenced from: /Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle/iOSInjection
  Reason: image not found
Please check this, Tks!

**Comment by TPQuietBro** [comment]
Xcode:9.3.1
Injection3

**Comment by johnno1962** [comment]
App store version?

**Comment by TPQuietBro** [comment]
yes,downloaded from AppStore.

**Comment by johnno1962** [comment]
Are you running Xcode from the default path /Applications/Xcode.app?

**Comment by TPQuietBro** [comment]
emmm,I ran Xcode not from /Applications/Xcode.app. I tried it as you said and it worked. Tks!

**Comment by mohamedkhedr** [comment]
Is it possible to run xcode from a different path and still work with the appstore version of injectionIII?
if not, where i have to change to point to my custom directory installation of xcode? Or i have to use a different approach?

**Comment by johnno1962** [comment]
No, you’d need to rebuild it from this repo: https://github.com/johnno1962/InjectionIII/tree/MAS-version

**Comment by mohamedkhedr** [comment]
thanks, changed the default path, exported and it's working as expected. maybe in the future i will add a directory selector in the menu, but it's fine for me right now.
